tests/function.test.js

```diff
@@ -0,0 +1,35 @@
+const describe = require('mocha').describe
+const it = require('mocha').it
+const expect = require('chai').expect
+const strictEqual = require('../function')
+
+
+describe('function', () => {
+  describe('strictEqual', () => {
+    it('returns true when given two parameters of the same type of value', () => {
+      // create test data 
+      const boolean = strictEqual(50, 50)
+      expect(boolean).to.equal(true)
+    })
+
+    it('returns false when given two parameters of the different type of value', () => {
+      //call the function with the test data 
+      const boolean = strictEqual('50', 50)
+      expect(boolean).to.equal(false)
+    })
+
+    it('returns false when given two parameters of the different type of value', () => {
+      //make assertions about what should be true 
+      const boolean = strictEqual(50, '3')
+      expect(boolean).to.equal(false)
+    })
```

Comment:
These two tests are effectively the same. They are both testing different type, different value. You should have four tests, one for each scenario

1. Same type, same value (ex. 50, 50)
2. Same type, different value (ex. 50, 60)
3. Different type, same value (ex. 50, '50')
4. Different type, different value (ex. 50, 'a')